mama/c_cpp/src/c/mama.c

```diff
@@ -235,6 +242,16 @@ mama_loadPayloadBridgeInternal  (mamaPayloadBridge* impl,
 mama_status
 mama_loadEntitlementBridgeInternal  (const char* name);
 
+void MAMACALLTYPE mamaImpl_entitlementDisconnectCallback (
```

Comment:
Actually I stand corrected on closer inspection it seems this function is only *called* in oea.c, so going to leave this as-is.